node/mock/requestedItemsHandlerStub.go

```diff
@@ -0,0 +1,23 @@
+package mock
+
+type RequestedItemsHandlerStub struct {
+	AddCalled   func(key string) error
+	HasCalled   func(key string) bool
+	SweepCalled func()
+}
+
+func (rihs *RequestedItemsHandlerStub) Add(key string) error {
```

Comment:
implement the functions to return default values if AddCalled, HasCalled, SweepCalled are nil.

Reply:
done